config.go

```diff
@@ -39,7 +39,7 @@ var DefaultConfig = Config{
 func MergeConfig(conf *Config) (*Config, error) {
 	merged := DefaultConfig
 	if conf != nil {
-		err := mergo.Merge(&merged, conf, mergo.WithOverride)
+		err := copier.Copy(&merged, conf)
```

Comment:
We would need to use `copier.CopyWithOption(&merged, conf, copier.Option{IgnoreEmpty: true})` to keep the behavior identical

Reply:
Since the from-struct (DefaultConfig) does not contain any nil values, there is nothing to ignore.
But I guess it won't hurt, especially if a new member is added to the Config struct and not set in the DefaultConfig.

Reply:
Actually from is the 2nd arg of Copy call which is `conf`, to is the 1st arg which is `DefaultConfig`, and typically `conf` contains lots of zero value, so it does make a difference.

Reply:
True, sorry, I've confused myself there.